### ISSUE TITLE
InGame Config not working

### ISSUE THREAD

**Issue by Chancho91** [mezz/JustEnoughItems#2398]
When I try to open the config, it says I need to install FTB Library, but I have it installed. I'm using the newest version 1.16.5 version of JEI.
![Screenshot (22)](https://user-images.githubusercontent.com/24527489/122333567-5fdcee80-cf06-11eb-85bb-93b0af821c10.png)

**Comment by YunanStratus** [comment]
I'm having the same issue.

**Comment by AffonsoPaulo** [comment]
Same issue here too.

**Comment by isaacrules2010** [comment]
glad i'm not the only one. :/

**Comment by Dragoreo** [comment]
looking at this issue might help.
https://github.com/mezz/JustEnoughItems/issues/1526
it's not solved but it is a known issue

**Comment by quasarea** [comment]
See https://github.com/mezz/JustEnoughItems/issues/2351, 
tldr downgrade jei to 7.6.4.90, they merged for FTB that were not in release yet

**Comment by AffonsoPaulo** [comment]
> See #2351,
> tldr downgrade jei to 7.6.4.90, they merged for FTB that were not in release yet

So, I tried this, but got an error where Applied requires version 7.7.0.98 or above. Is there another way?
